angular/official/list/src/app/view-message/view-message.page.ts

```diff
@@ -1,31 +1,28 @@
-import { CommonModule } from '@angular/common';
-import { Component, inject, OnInit } from '@angular/core';
+import { Component, OnInit } from '@angular/core';
 import { ActivatedRoute } from '@angular/router';
-import { IonicModule, Platform } from '@ionic/angular';
 import { DataService, Message } from '../services/data.service';
 
 @Component({
   selector: 'app-view-message',
   templateUrl: './view-message.page.html',
   styleUrls: ['./view-message.page.scss'],
-  standalone: true,
-  imports: [IonicModule, CommonModule],
 })
 export class ViewMessagePage implements OnInit {
   public message!: Message;
-  private data = inject(DataService);
-  private activatedRoute = inject(ActivatedRoute);
-  private platform = inject(Platform);
 
-  constructor() {}
+  constructor(
+    private data: DataService,
+    private activatedRoute: ActivatedRoute
+  ) { }
 
   ngOnInit() {
     const id = this.activatedRoute.snapshot.paramMap.get('id') as string;
     this.message = this.data.getMessageById(parseInt(id, 10));
   }
 
   getBackButtonText() {
-    const isIos = this.platform.is('ios')
-    return isIos ? 'Inbox' : '';
+    const win = window as any;
```

Comment:
Can we keep the previous code for the `isIos` check here (adding `platform` to the constructor + import).